install-storageos/020-install-storageos.md

```diff
@@ -0,0 +1,100 @@
+
+First we need to install the StorageOS Operator using the following yaml manifest.
+
+`kubectl create -f https://github.com/storageos/cluster-operator/releases/download/v2.1.0/storageos-operator.yaml`{{execute}}
+
+Verify the Cluster Operator pod status
+
+`kubectl -n storageos-operator get pod -w`{{execute}}
```

Comment:
Depending on how fast you click, if the operator pod is already running, this blocks indefinitely. Perhaps combine with the above command (&&) ?